src/collections/_documentation/platforms/javascript/sourcemaps/troubleshooting.md

```diff
@@ -5,11 +5,17 @@ sidebar_order: 3
 
 Source maps can sometimes be tricky to get going. If you’re having trouble, try the following tips.
 
-### Verify your source maps are built correctly
+### Verify release is configured
```

Comment:
Could this be "Verify release is configured in your SDK"?

You're also (IMHO) bringing up two different problems here - not having included a release value in your SDK config at all, and having included a release value in your SDK which doesn't match the name of the release you created with the CLI ("... and not have tagged your errors with the _same_ release").

I'd be tempted to make the first paragraph here read: 

"In order for sourcemaps to work, the release needs to be created by the CLI (and the correct files associated with it), _and_ the name of that newly-created release needs to be included in your SDK configuration." 

and then make a separate section basically saying "the values have to match, dummy. here's how to check."